ui/raidboss/data/05-shb/dungeon/qitana_ravel.js

```diff
@@ -15,24 +15,7 @@
       regexJa: Regexes.startsUsing({ id: '3C89', source: 'ロツァトル' }),
       regexCn: Regexes.startsUsing({ id: '3C89', source: '洛查特尔' }),
       regexKo: Regexes.startsUsing({ id: '3C89', source: '로차틀' }),
-      alertText: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            ko: '나에게 탱버',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            ko: data.ShortName(matches.target) + '에게 탱버',
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
Could you add a condition here of "target is you or role is healer" to preserve the previous logic?